examples/sdk-examples/src/examples/chartResponsiveness/index.tsx

```diff
@@ -5,19 +5,28 @@ import React from "react";
 import { ExampleWithSource } from "../../components/ExampleWithSource";
 import { HeadlineResponsiveExample } from "./HeadlineResponsiveExample";
 import { BarChartResponsiveExample } from "./BarChartResponsiveExample";
+import { ColumnChartResponsiveLegendExample } from "./ColumnChartResponsiveLegendExample";
 
 import HeadlineResponsiveExampleSrc from "!raw-loader!./HeadlineResponsiveExample";
 import BarChartResponsiveExampleSrc from "!raw-loader!./BarChartResponsiveExample";
+import ColumnChartResponsiveLegendExampleSrc from "!raw-loader!./ColumnChartResponsiveLegendExample";
 
 import HeadlineResponsiveExampleSrcJS from "!raw-loader!../../../examplesJS/chartResponsiveness/HeadlineResponsiveExample";
 import BarChartResponsiveExampleSrcJS from "!raw-loader!../../../examplesJS/chartResponsiveness/BarChartResponsiveExample";
+import ColumnChartResponsiveLegendExampleSrcJS from "!raw-loader!../../../examplesJS/chartResponsiveness/ColumnChartResponsiveLegendExample";
 
 export const ChartResponsiveness: React.FC = () => (
     <div>
         <h1>Chart Responsiveness</h1>
-        <p>These examples show improved responsive behaviour of charts.</p>
+        <p>These examples show improved responsive behaviour of charts and headline.</p>
```

Comment:
These examples show responsive behavior of charts.

Reply:
I suggest that we do not mention "improved" - compared to what? Also, let's not specify headlines as a separate category. Having it as a separate example makes it stand out sufficiently.